ccd-claim-migration/build.gradle

```diff
@@ -0,0 +1,35 @@
+apply plugin: 'java'
```

Comment:
@rlewan  will we need to do anything to make sure this whole submodule is excluded from sonar?

Reply:
@rlewan  this is my only worry left on this PR atm